Documentation/manualtest-guide.md

```diff
@@ -1,17 +1,100 @@
 Manual Tests Instruction
 ========================
-Test Name: WindowsAuthentication_RoundTrips_Echo
 
-Set Up Requirement
+**Windows: Windows Authentication tests**
+
+Tests:  
+```csharp
+    WindowsAuthentication_RoundTrips_Echo
+```
+
+Requirements:
 
 1. Must run on a domain machine.
 
-Steps
+Steps:
 
 1. Enable the test
-In src\System.Private.ServiceModel\tests\Scenarios\Security\TransportSecurity\Https\ClientCredentialTypeTests.cs,
+in src\System.Private.ServiceModel\tests\Scenarios\Security\TransportSecurity\Https\ClientCredentialTypeTests.cs,
 Remove  [ActiveIssue(53)] above test method WindowsAuthentication_RoundTrips_Echo()
 2. Go to the root of WCF source, run all scenario tests.
 ```
-	     build /p:WithCategories=OuterLoop
+    .\build.cmd /p:WithCategories=OuterLoop
+```
+
+
+**Linux: HTTPS tests** 
+
+Tests:
+```csharp
+   HttpsTests.SameBinding_Soap12_EchoString 
+   HttpsTests.CrossBinding_Soap11_EchoString
+   HttpsTests.SameBinding_Soap11_EchoString 
+   HttpsTests.ServerCertificateValidation_EchoString
+   Https_ClientCredentialTypeTests.BasicAuthenticationInvalidPwd_throw_MessageSecurityException
+   Https_ClientCredentialTypeTests.BasicAuthentication_RoundTrips_Echo
+```
+
+The WCF Test Root CA certificate need to be retrieved from the Bridge and installed on the local machine before tests can be run
+
+Requirements: 
+
+* Requires a Windows machine running the Bridge  
+
+Assumes: 
+
+* $(GitWcfRoot) is the root of the WCF Core git repo
+* $(BridgeHost) is the Bridge host machine name  
+
+Steps:
+
+1. Ensure the Bridge is running on a Windows machine with remote access allowed: `startBridge.cmd -allowRemote`
```

Comment:
Hmm, I guess I've assumed the test and server machines are on the same subnet, so this wasn't an issue. I'll add this.